pkg/jerryscript/Makefile.jerryscript

```diff
@@ -31,7 +31,7 @@ all: libjerry
 libjerry:
 	cmake -B$(JERRYSCRIPT_BUILD_DIR) -H./ \
 	 -DCMAKE_SYSTEM_NAME=RIOT \
-	 -DCMAKE_SYSTEM_PROCESSOR="$(MCPU)" \
+	 -DCMAKE_SYSTEM_PROCESSOR:FORCE="$(MCPU)" \
```

Comment:
Why did you need to add FORCE?

Reply:
Ah, I forgot to remove this... It's not needed, I'll remove it.

Reply:
Ok, squash afterwards!

Reply:
ups there was nothing to squash.

Reply:
Wait, I still have to push the update ;)

Reply:
Done